src/main/java/org/bankofspring/model/AccountTransaction.java

```diff
@@ -0,0 +1,136 @@
+package org.bankofspring.model;
+
+import java.io.Serializable;
+import java.util.Calendar;
+import java.util.GregorianCalendar;
+import java.util.TimeZone;
+
+/**
+ * Class to model an account transaction. If it is a debit, the fromAccount will
+ * be populated and the amount will be negative. If it is a credit the toAccount
+ * will be populated and the amount will be positive. The other account can be
+ * populated if the transaction represents a payment or transfer.
+ * 
+ * @author ram
+ *
+ */
+public class AccountTransaction implements Serializable {
+
+	private static final long serialVersionUID = 9011178094204166131L;
+
+	private Calendar txnDate;
+
+	private Account fromAccount;
+
+	private Account toAccount;
+
+	private Long amount;
+
+	/**
+	 * Constructor.
+	 * 
+	 * If this transaction represents a debit, the fromAccount will be populated
+	 * and the amount will be negative. If it is a credit the toAccount will be
+	 * populated and the amount will be positive. The other account can be
+	 * populated if the transaction represents a payment or transfer.
+	 * 
+	 * @param fromAccount
+	 * @param toAccount
+	 * @param amount
+	 */
+	public AccountTransaction(Account fromAccount, Account toAccount,
+			Long amount) {
+
+		// Works out the date and time in GMT
+		this.txnDate = GregorianCalendar.getInstance(TimeZone
+				.getTimeZone("GMT"));
```

Comment:
Apparently you should use the full name rather than the 3 digits as some some digits have multiple meanings (e.g. BST = British Summer Time and Bangladesh Standard Time).